### ISSUE TITLE
Removing 'colors' from auth0_branding breaks Universal Login branding

### ISSUE THREAD

**Issue by jeffreymlewis** [auth0/terraform-provider-auth0#643]
### Checklist

- [X] I have looked into the [README](https://github.com/auth0/terraform-provider-auth0#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [documentation](https://registry.terraform.io/providers/auth0/auth0/latest/docs) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/terraform-provider-auth0/issues) and have not found a suitable solution or answer.
- [X] I have upgraded to the [latest version](https://github.com/auth0/terraform-provider-auth0/releases/latest) of this provider and the issue still persists.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

Removing the `colors` block from the `auth0_branding` resource causes Auth0 to revert to the default login page. The Universal Login preview in the console still looks correct. It's necessary to have a _browser_ actually hit the auth0 login page to see this behavior.

### Expectation

After running terraform, I expect the login page to render correctly.

### Reproduction

The reproduction steps are difficult to explain. I'll do my best!

1. setup Universal login branding using the console: Branding > Universal Login > Customization Options (`/universal-login/customizations/colors`)
2. setup a terraform module using the `auth0_branding` and `auth0_branding_theme` resources.
3. run `terraform import` to import the existing configuration into the terraform remote state
4. run `terraform plan` and note no changes
5. add a `colors` block to the `auth0_branding` resource and run `terraform apply`.
6. hit the login page with a _browser_ and observe that it's still working.
7. remove `colors` block from the `auth0_branding` resource and run `terraform apply`
8. Hit the login page with a _browser_ and observe it's reverted to default. Note the Universal Login preview in the Auth0 console is still correct. **It's necessary to have an app which can actually hit the Auth0 login page to observe this behavior.**
9. add the `colors` block back to the `auth0_branding` resource, run terraform, and note that the login page is still broken
10. make a trivial change to Universal Login customizatoin via the console (Ex. change one color from #FFFFFF to #FFFFFE) and apply. Note that the login page is now correct.

Note that a JSON dump of the tenant before and after step 7 and step 10 shows no difference.  (`a0deploy export ...`)

### Auth0 Terraform Provider version

v0.49.0

### Terraform version

v1.4.6

**Comment by sergiught** [comment]
Hey @jeffreymlewis 👋🏻 

Thanks for taking the time to raise this with us. We'll look into this and come back with an update soon. Appreciate your patience!

**Comment by sergiught** [comment]
Hey @jeffreymlewis 👋🏻 

We've done some changes within the `auth0_branding` resource in the https://github.com/auth0/terraform-provider-auth0/releases/tag/v0.50.0 that could fix your issue, mind giving it a try? Also to note tho that having a theme applied, will effectively nulify any colors set through the `auth0_branding` resource, that might also be what happened in your case.

**Comment by sergiught** [comment]
Hey @jeffreymlewis   👋🏻 

This should be fixed now as part of the [v1.0.0-beta.0](https://registry.terraform.io/providers/auth0/auth0/latest/docs/resources/log_stream#filters) release! 🙌🏻 Please give it a try and let us know if you encounter any issues.

```
terraform {
  required_providers {
    auth0 = {
      source  = "auth0/auth0"
      version = "1.0.0-beta.0"
    }
  }
}
```

### 📚 Resources

[Quickstart](https://registry.terraform.io/providers/auth0/auth0/latest/docs/guides/quickstart) | [Documentation](https://registry.terraform.io/providers/auth0/auth0/latest/docs) | [Migration Guide](https://github.com/auth0/terraform-provider-auth0/blob/v1/MIGRATION_GUIDE.md) | [Changelog](https://github.com/auth0/terraform-provider-auth0/releases/tag/v1.0.0-beta.0)
